magda-registry-api/src/main/scala/au/csiro/data61/magda/registry/RegistryApp.scala

```diff
@@ -60,7 +60,7 @@ object RegistryApp extends App {
   logger.info("Starting MAGDA Registry")
 
   GlobalSettings.loggingSQLAndTime = LoggingSQLAndTimeSettings(
-    enabled = false,
+    enabled = true,
```

Comment:
Will this print SQL for every requests? Do we want this for production?

Reply:
Oops